.prettierignore

```diff
@@ -1 +1,2 @@
 /.yarn
+/src/abis
```

Comment:
nitpick: newline a the end of file

Reply:
Inconsequential as of 3ee2bbeb0c6ad411221151d30b9ead8cc3331cef.